packages/devtools_app/lib/src/screens/memory/panes/diff/diff_pane.dart

```diff
@@ -62,13 +63,20 @@ class _SnapshotItemContent extends StatelessWidget {
               crossAxisAlignment: CrossAxisAlignment.start,
               children: [
                 Expanded(
-                  child: Markdown(
-                    data: _snapshotDocumentation,
-                    onTapLink: (text, url, title) async =>
-                        await launchUrl(url!),
+                  child: ListView(
+                    children: [
+                      SizedBox(
```

Comment:
instead of wrapping in a ListView and a SizedBox, why not just put the `ClassTypeLegend` in the parent column before the `MoreInfoLink`?

Reply:
Because we want it to be scrolled together with the main content. So, it should be in ListView, that provides scrolling.